src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/RazorFormattingTest.cs

```diff
@@ -705,6 +705,31 @@ await RunFormattingTestAsync(
             fileKind: FileKinds.Legacy);
     }
 
+    [Fact]
+    public async Task MultiLineComment_WithinHtml ()
+    {
+        await RunFormattingTestAsync(
+            input: """
+                    <div>
+                        @*
+                            <div>
+                                This comment's indentation will be preserved
```

Comment:
Nit, or possibly paranoia, but _just in case_ perhaps it would be good for the comment contents to not be indented perfectly. If we accidentally regress this in future, but also make `@*` indent its contents, this test may not fail. Even something as simple as this should do it:

```suggestion
    This comment's indentation will be preserved
```

Ugly, yes, but it's a test so it's okay :)